WorkingGroups/NameChange/2022-04-08.md

```diff
@@ -0,0 +1,105 @@
+# ISIS Name Change Working Group Meeting April 28, 2022 @1PM MST / 4PM EDT
+
+### Attending
+
+- Jesse Mapel
+- Lauren Adoram-Kershner
+- Audrie Fennema
+- Kestay Laszlo
+- Lori Pigue
+- Stuart Robbins
+- Victor Silva
+
+## Agenda / Notes
+
+- Introduction
+  - Expectations
+    - Do not share the raw data
+      - Victor will share an edited version with some LRO folks
+    - All these notes will be public
+    - Go over things outside of meetings
+    - Meetings are for discussion
+  - Goals
+    - Proposal to be present to the ISIS TC and the ASC management (and the public)
+    - Maybe we need to make another survey
+    - Form the large amount of differing opinions, perspectives, and stakeholders into something usable
+    - What needs to happen to make a name change successful (if we decide it should happen)
+  - Timeline
+    - Proposal scheduled June 9th 1PM Pacific/MST
+      - Date is good
+    - Future meetings
+      - Expect at least one more working group meeting is required
+      - 6 weekly meetings seems like too much
+      - Schedule meeting as we go based on required work
+      - Table to end of meeting
+  - Collaborating
+    - ASC folks have access to shared excel forms
+    - We could use a google sheet from a personal e-mail
```

Comment:
Please do not use a google sheet if this is USGS led. Google sheets are not a DOI approved alternative. You can and should use Sharepoint. It is possible to share those documents with external collaborators.

Reply:
Can we use Microsoft Teams? I work for another DOI entity and we use Teams for sharing/collaborating documents with external collaborators.

Reply:
Absolutely! Sharepoint is the MS approved collaboration tool. I have not tested Teams-Sharepoint integration but I assume they work really well since they are both MS tools.

Reply:
I've tried to use Sharepoint and I've uploaded it via teams, but I don't think everyone in the WG has either. Sharepoint does not permit edit access to anyone outside the DOI and that is what I need. @jlaura Is there any way around this?

Reply:
Have you tried to share individually with each person on the WG using their email address?

Reply:
When I attempt to share with an e-mail address through outlook, excel, and sharepoint is says edit access cannot be given to users outside the DOI. I even got a nice e-mail error message back from outlook.

Reply:
Do we need to change this in the notes to reflect how we actually shared the notes?

Reply:
No, I'll update in the next meeting notes